modules/crypt.cpp

```diff
@@ -43,16 +42,151 @@
 #include <znc/Chan.h>
 #include <znc/User.h>
 #include <znc/IRCNetwork.h>
+#include <openssl/dh.h>
+#include <openssl/bn.h>
+#include <znc/SHA256.h>
 
 #define REQUIRESSL 1
 #define NICK_PREFIX_KEY "[nick-prefix]"
 
 class CCryptMod : public CModule {
+  private:
+    const char *prime1080="FBE1022E23D213E8ACFA9AE8B9DFADA3EA6B7AC7A7B7E95AB5EB2DF858921FEADE95E6AC7BE7DE6ADBAB8A783E7AF7A7FA6A2B7BEB1E72EAE2B72F9FA2BFB2A2EFBEFAC868BADB3E828FA8BADFADA3E4CC1BE7E8AFE85E9698A783EB68FA07A77AB6AD7BEB618ACF9CA2897EB28A6189EFA07AB99A8A7FA9AE299EFA7BA66DEAFEFBEFBF0B7D8B";
```

Comment:
Used in all other DH1080 fish implementations I could find (in either hex, dec or base64 encoded form): [KVIrc](https://github.com/Cizzle/KVIrc/blob/master/src/modules/fish/libkvifish.cpp#L70), [fish10](https://github.com/flakes/mirc_fish_10/blob/master/fish_10/src/dh1080.cpp#L57), ZNC FiSH [1 L118](http://pastebin.com/K8zvJ89v) and [2 L114](http://slexy.org/view/s2poFq0BaF), [FiSH-irssi](https://github.com/falsovsky/FiSH-irssi/blob/master/src/DH1080.c#L15), [Quassel IRC](https://github.com/quassel/quassel/blob/master/src/core/cipher.cpp#L20), [XChat-FiSH-AES](https://github.com/emigonza/XChat-FiSH-AES/blob/master/XChat_AES.py#L272) and so on...
Also mentioned [here](http://blog.bjrn.se/2009/01/proposal-for-better-irc-encryption.html).